lua/autosave/modules/autocmds.lua

```diff
@@ -30,7 +30,14 @@ local function actual_save()
     -- might use  update, but in that case it can't be checekd if a file was modified and so it will always
     -- print opts["execution_message"]
     if (api.nvim_eval([[&modified]]) == 1) then
+        local first_char_pos = vim.fn.getpos("'[")
+        local last_char_pos = vim.fn.getpos("']")
+
         cmd("silent! write")
+
+        vim.fn.setpos("'[", first_char_pos)
+        vim.fn.setpos("']", last_char_pos)
```

Comment:
To avoid lsp warnings about `vim` being an undefined global and to increase readability we could use `fn` which is set at the top of the file instead of `vim.fn` all the time.

Reply:
Thanks for the PR!